gnd/src/main/java/com/google/android/gnd/ui/recorddetails/RecordDetailsFragment.java

```diff
@@ -66,13 +60,24 @@
   @Override
   public void onCreate(@Nullable Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
+    RecordDetailsFragmentArgs args = getRecordDetailFragmentArgs();
     viewModel = getViewModel(RecordDetailsViewModel.class);
+    viewModel.loadRecordDetails(args);
```

Comment:
This should be the last thing we do here to avoid race condition w/observe. Technically if the LiveData are backed by BehaviorSubjects it won't matter, but we don't know that's what's used here, so better to be safe.

Reply:
got it!